modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/java/com/liferay/dynamic/data/mapping/form/report/web/internal/display/context/DDMFormReportDisplayContext.java

```diff
@@ -189,6 +193,39 @@ private JSONObject _getDDMFormFieldOptionLabelsJSONObject(
 		return jsonObject;
 	}
 
+	private String _getLabel(DDMFormField ddmFormField) {
+		if (StringUtil.equals(
+				ddmFormField.getType(),
+				DDMFormFieldTypeConstants.SEARCH_LOCATION)) {
+
+			JSONObject jsonObject = JSONFactoryUtil.createJSONObject();
+
+			LocalizedValue localizedValue =
+				(LocalizedValue)ddmFormField.getProperty("visibleFields");
```

Comment:
@marcelabc, shouldn't we rename the variable to `visibleFields` instead?

Actually, this [commit](https://github.com/liferay-data-engine/liferay-portal/pull/567/commits/fea4ad9845d62f7875a1ad26cd8ee5abc23300f7#) of this pull named a similar a variable as `visibleFields`. In light of this, can we rename the 2 variables consistently?